src/olympia/files/models.py

```diff
@@ -116,7 +116,7 @@ def from_upload(cls, upload, version, parsed_data=None):
         assert parsed_data is not None
 
         file_ = cls(version=version)
-        upload_path = force_str(nfd_str(upload.path))
```

Comment:
Drive-by change, but `upload.path` nowadays only contains ascii - plus we're not running Python 2 anymore, unicode paths just work.